public/js/app.js

```diff
@@ -39043,16 +39043,15 @@ var Header = function Header(props) {
     var uriName = props.cookies.get("activePage");
 
     if (uriName !== undefined) {
-      if (uriName === "/dashboard") {
-        setURI(uriName);
-      }
+      switch (uriName) {
+        case "/dashboard":
+          setURI(uriName);
```

Comment:
put `break` after every `case`
https://www.w3schools.com/js/js_switch.asp

Reply:
This is resolved ma'am.